src/main/kotlin/com/noumenadigital/npl/cli/commands/registry/VersionCommand.kt

```diff
@@ -0,0 +1,15 @@
+package com.noumenadigital.npl.cli.commands.registry
+
+import java.io.Writer
+
+class VersionCommand : NplCommandExecutor {
+    companion object {
+        const val COMMAND_DESCRIPTION = "Command to return current npl cli version"
```

Comment:
Nit:

I would go with

```suggestion
        const val COMMAND_DESCRIPTION = "Display the current version of the NPL CLI"
```

and similar for other commands. That is:

- use imperative mood
- drop the "Command to..."

Reply:
fixed